### ISSUE TITLE
ReferenceError: mixin is not defined

### ISSUE THREAD

**Issue by frostbite49er** [avocode/avocode#1008]
IMPORTANT: Please try to describe how did it happen.

-----
App v.2.8.4
Source UID: 82138
```
ReferenceError: mixin is not defined
    at declareAbsolutePosition (C:\Users\49er\AppData\Local\avocode\app-2.8.4\resources\app.asar\node_modules\csshat-language-sass\main.js:98:14)
    at Sass.render (C:\Users\49er\AppData\Local\avocode\app-2.8.4\resources\app.asar\node_modules\csshat-language-sass\main.js:215:11)
    at e.build (file:///C:/Users/49er/AppData/Local/avocode/app-2.8.4/resources/app.asar/dist/avocode.min.js:2:18068)
    at e._classRenderer (file:///C:/Users/49er/AppData/Local/avocode/app-2.8.4/resources/app.asar/dist/avocode.min.js:2:16899)
    at e.<anonymous> (file:///C:/Users/49er/AppData/Local/avocode/app-2.8.4/resources/app.asar/dist/avocode.min.js:2:15376)
    at e.render (file:///C:/Users/49er/AppData/Local/avocode/app-2.8.4/resources/app.asar/dist/avocode.min.js:2:17094)
    at file:///C:/Users/49er/AppData/Local/avocode/app-2.8.4/resources/app.asar/dist/avocode.min.js:2:14906
    at C:\Users\49er\AppData\Local\avocode\app-2.8.4\resources\app.asar\node_modules\a
```

**Comment by petrbrzek** [comment]
Thanks for reporting the issue. It will be fixed in the upcoming release.

**Comment by petrbrzek** [comment]
Fixed in 2.9.0.
